foundations/html_css/html-foundations/links-and-images.md

```diff
@@ -172,12 +172,36 @@ For example, using an absolute path we can display an image located on The Odin
 
 To use images that we have on our own websites, we can use a relative path.
 
+<details markdown="block">
+<summary class="dropDown-header">Linux, macOS, ChromeOS
+</summary>
+
 1.  Create a new directory named `images` within the `odin-links-and-images` project.
 
 2.  Next, download [this image](https://unsplash.com/photos/Mv9hjnEUHR4/download?force=true&w=640) and move it into the images directory we just created.
 
 3.  Rename the image to `dog.jpg`.
 
+</details>
+
+<details markdown="block">
+<summary class="dropDown-header">WSL
+</summary>
+
+When you download a file from the internet, Windows has a security feature that creates a hidden `Zone.Identifier` file with the same name as your downloaded file and it looks like `mypicture.jpg:Zone.Identifier` This file is harmless, but we'd like to avoid copying it over and cluttering up our directories.
+
+1.  Create a new directory named `images` within the `odin-links-and-images` project.
+
+2.  Next, download [this image](https://unsplash.com/photos/Mv9hjnEUHR4/download?force=true&w=640).
+
+3.  Right click on the new download at the bottom of the chrome window and select "Show in folder".
+
+4.  Drag the file from your downloads folder to VSCode's file browser into your new `images` directory.
+
+Dragging files from Windows into the VSCode file browser prevents the `Zone.Identifier` files from being copied over. From now on, any time you need to copy pictures or other downloaded files like this into WSL, you can do it in this way. If you ever accidentally copy these `Zone.Identifier` files into WSL, you can safely delete them without any issue.
```

Comment:
I feel like we should still include some alternative instructions to drag and drop, even it means warning that copy pasting or whatever other way may result in a (harmless) duplicate file.

Reply:
There's a small amount of dragging and dropping with the alternative:

- Using your Ubuntu terminal, navigate to the folder you want to copy the image to (`cd ~/odin-links-and-images` for example)
- Type `cp <space>`
- Drag the dog.jpg image from a Windows Explorer window and drop it onto the terminal window, it should appear as `"/mnt/c/users/username/Downloads/dog.jpg"`
- Type `<space> .` to tell `cp` that you want to copy the file to your current working directory
  -  The full command will look something like `cp "/mnt/c/users/username/Downloads/dog.jpg" .`
- Hit <enter> to complete the command, and use `ls` to confirm the file now exists

Reply:
Maybe the solution is as simple as updating the verbiage to "Either drag and drop or copy and paste..." or something

Reply:
Added the extra explanation by @xandora , which I think should be enough. If we see that lots of learners on Discord run into issues at this step we can always add extra instructions